browser.c

```diff
@@ -1606,21 +1606,60 @@ void mutt_buffer_select_file(struct Buffer *file, SelectFileFlags flags,
           mutt_message("%s", ARRAY_GET(&state.entry, menu->current)->name);
         break;
 
-#ifdef USE_IMAP
-      case OP_BROWSER_TOGGLE_LSUB:
-        bool_str_toggle(NeoMutt->sub, "imap_list_subscribed", NULL);
+      case OP_CREATE_MAILBOX:
```

Comment:
Is there a preference for this code to be moved to a separate function instead?

Reply:
yes, good idea